frontend/src/components/UI/Cards/GitProviderTabCard.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
I think you can remove the file `GitProviderTabCard` from the commit `fix update modal not opening` :)

Reply:
done